app/frontend/src/components/SupportingContent/SupportingContent.tsx

```diff
@@ -6,32 +6,23 @@ interface Props {
     supportingContent: string[] | { text: string[]; images?: { url: string }[] };
 }
 
-interface SupportingItemProps {
-    title: string;
-    content: string;
-}
-
 export const SupportingContent = ({ supportingContent }: Props) => {
     const textItems = Array.isArray(supportingContent) ? supportingContent : supportingContent.text;
     const imageItems = !Array.isArray(supportingContent) ? supportingContent?.images : [];
     return (
         <ul className={styles.supportingContentNavList}>
-            {textItems.map(c => {
+            {textItems.map((c, ind) => {
```

Comment:
Ditto, this was just fixing index. And I moved the HTML inside the class as I couldn't deal with writing the correct Typescript for the separate component. :)